### ISSUE TITLE
Update links point to IDPF that should point to W3C

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#112]
In the introduction, there is a link to:
http://idpf.org/epub/a11y/#sec-disc-package

I think this should point to:
https://www.w3.org/Submission/epub-a11y/#sec-disc-package

I wanted to check with Matt if this should be changed.

Also, now that The EPUB Accessibility spec 1.1 is in CR, should we be using that link? Also, would that link resolve properly once it reaches REC status?
